src/python/build_venv.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+TARGET_DIR=$1
+JAIA_DIR="$(pwd)/../../"
+
+# Build the venv
+echo 🟢 Building the python venv into ${TARGET_DIR}
+
+## Make target python directory
+mkdir -p ${TARGET_DIR}
```

Comment:
It needs to create all parent directories to the path, in case they don't exist, but...  now that I think of it, it might be a problem if the venv already exists and we want to clobber it, (removing all the installed packages).

Reply:
If we want to clobber any pre-existing venv directories, that would be an additional "rm -rf ${TARGET_DIR}" just before the mkdir.